### PR TITLE
fix(RELEASE-1069): fix linting issues in embargo-check task

### PR DESCRIPTION
**Pull request by jinqi7**

### CODE REVIEW

**Review by johnbieren — APPROVED** [review]

File: tasks/embargo-check/embargo-check.yaml [hunk 1]
```diff
@@ -59,12 +59,12 @@ spec:
         RC=0
 
         NUM_ISSUES=$(jq -cr '.releaseNotes.issues.fixed | length' "${DATA_FILE}")
-        for ((i = 0; i < $NUM_ISSUES; i++)); do
+        for ((i = 0; i < NUM_ISSUES; i++)); do
```
Comment (scoheb) [inline tasks/embargo-check/embargo-check.yaml:62]: missing `$`

Comment (jinqi7) [reply tasks/embargo-check/embargo-check.yaml:62]: With "$" added, checkton reports the warning: $/${} is unnecessary on arithmetic variables. It's a fix for the warning.

Comment (scoheb) [reply tasks/embargo-check/embargo-check.yaml:62]: ok!

**Review by scoheb — APPROVED** [review]

**Pull request merged**